Makefile

```diff
@@ -58,9 +61,19 @@ build-android:
 		go build -v -tags no_cgo \
 		-o $(OUTPUT) ./cmd/module/cmd.go
 
-module.tar.gz: $(OUTPUT) run.sh
-	# todo: dedup with 'make module' command
-	tar czf $@ $^ -C $(FFMPEG_PREFIX) lib
+.PHONY: module
+ifeq ($(GOOS),android)
```

Comment:
@abe-winter There is probably a better way to do this but current workaround is simple if check on GOOS.

- If Android, copy over binary, run.sh, and ffmpeg libs
- If Linux, copy over AppImg and run.sh